synapse/storage/databases/main/monthly_active_users.py

```diff
@@ -328,15 +334,15 @@ def upsert_monthly_active_user_txn(self, txn, user_id):
             values={"timestamp": int(self._clock.time_msec())},
         )
 
-        self._invalidate_cache_and_stream(txn, self.get_monthly_active_count, ())
-        self._invalidate_cache_and_stream(
+        self._invalidate_cache_and_stream(txn, self.get_monthly_active_count, ())  # type: ignore[attr-defined]
```

Comment:
I think some more of these ignores are redundant after 3a7ef03b1f9b894a29820ba65e0701c4f1b97cb7. Try running `mypy --warn-unused-ignores synapse/storage/databases/main`!